src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: "20px 0",
+    lineHeight: "1.5em",
+  }
+  return (
+    <header style={headerStyle}>
+      <h1  style={{
+        fontSize: "6rem",
+        fontWeight: "600",
+        marginBottom: "2rem",
+        lineHeight: "1em",
+        color: "#ececec",
+        textTransform: "lowercase",
+        textAlign: "center",
+      }}>To do List</h1>
```

Comment:
Hello @AbbyNyakara  , since we are using dry you can store some of the css attributes as variables , so you can reuse them